pulp_rpm/app/serializers.py

```diff
@@ -337,37 +337,46 @@ class UpdateRecordSerializer(NoArtifactContentSerializer):
     """
 
     id = serializers.CharField(
-        help_text=_("Update id (short update name, e.g. RHEA-2013:1777)")
+        help_text=_("Update id (short update name, e.g. RHEA-2013:1777)"),
+        allow_blank=True
     )
     updated_date = serializers.CharField(
-        help_text=_("Date when the update was updated (e.g. '2013-12-02 00:00:00')")
+        help_text=_("Date when the update was updated (e.g. '2013-12-02 00:00:00')"),
+        allow_blank=True
```

Comment:
Updated_date has to be present until we add support for opensuse advisories which might not have it at all